angular-primeng-app/src/app/models/post.ts

```diff
@@ -47,3 +47,12 @@ export interface Content {
   html: string;
 }
 
+export interface PostsPageInfo {
+  posts: Post[];
+  pagination: PageInfo;
+}
+
+export interface PageInfo {
+  hasNextPage: boolean;
+  endCursor: string;
+}
```

Comment:
@czBalazs98 just checked it out, works great and 
I like the interfaces like this 👍 I will probably make 
the other versions match  - in this case it's better to 
add additional interfaces instead of changing the original

cool :)